fbw-a32nx/src/systems/fmgc/src/guidance/vnav/VnavDriver.ts

```diff
@@ -195,6 +206,8 @@ export class VnavDriver implements GuidanceComponent {
       this.oldLegs.clear();
       this.guidanceController.pseudoWaypoints.acceptVerticalProfile();
       this.previousManagedDescentSpeedTarget = undefined;
+
+      SimVar.SetSimVarValue('L:A32NX_FM_VERTICAL_PROFILE_AVAIL', 'Bool', false);
```

Comment:
Actually, you probably also want to set `prevMcduPredReadyToDisplay` to false here to prevent it from being out of sync with the LVar.

Reply:
Good point, should be fixed now